docs/develop/feel-expression.md

```diff
@@ -138,13 +138,29 @@ duration("P1Y") / duration("P1M")
 | greater than or equal | `>=` | `>= 10` |
 | between | `between _ and _` | `x between 3 and 9` |
 
-* less than/greater than/between are only supported for: 
+The operators less than, greater than, and between are only supported for: 
   * number
   * date
   * time
   * date-time
   * year-month-duration
   * day-time-duration 
+  
+Any value can be compared with `null` to check if it is equal to `null`, or if it exists. Comparing `null` to a value different from `null` results in `false`. It returns `true` if the value, or the property is `null` or doesn't exist. 
```

Comment:
Maybe add a reference to "is defined()" here and explain that this is a more precise check for existence